src/app_selection.rs

```diff
@@ -128,7 +128,7 @@ mod tests {
     use std::str::FromStr;
     use tokio::sync::RwLock;
 
-    fn new_fake_app_info(app_id: impl ToString, dna_hash: DnaHash) -> AppInfo {
+    pub fn new_fake_app_info(app_id: impl ToString, dna_hash: DnaHash) -> AppInfo {
```

Comment:
Exposing this to be re-used. Could be moved to some shared test module.

Reply:
Agree, we have a few things we should do that with, same with the app info creation